run_interactive.py

```diff
@@ -40,7 +40,8 @@ def main(argv):
   max_output_length = 1024
 
   profiling_output = FLAGS.profiling_output
-  if profiling_output:
+  profiling_prefill = FLAGS.profiling_prefill
```

Comment:
Current logic only handles:
1. profile both output and prefill
2. profile output but not prefill

If profile prefill but not output, the current logic won't profile at all. Not sure if you want to handle this case as well. We can add in future PR if there is such use case as well.